client/scss/layouts/_login.scss

```diff
@@ -65,6 +65,11 @@
       max-width: calc(theme('spacing.32') * 3.5);
       padding: theme('spacing.12') theme('spacing.14');
     }
+
+    // border for windows high contrast mode, light
+    @media (forced-colors: active) {
+      border: 5px solid transparent;
+    }
```

Comment:
@Lovelyfin00  it looks like some other changes from other PRs have leaked into this PR - please can you clean this up